### ISSUE TITLE
val_boxes.json and test_boxes.json question

### ISSUE THREAD

**Issue by AdamWP** [TensorBox/TensorBox#177]
I notice that in the brainwash dataset there are three sets of images; training, validation, and test.  Each has a corresponding json file.  However, hypes/overfeat_rezoom.json contains only the training and validation files (lines 3 & 4).   The README.md mentions using val_boxes.json for evaluation evaluate.py.  "python evaluate.py --weights output/overfeat_rezoom_2017_01_17_15.20/save.ckpt-130000 --test_boxes data/brainwash/val_boxes.json"

Should hypes/overfeat_rezoom.json specify test_boxes.json on line 4?

If not, what is the purpose of the test dataset?

**Comment by Russell91** [comment]
The purpose of the test dataset is for testing the quality of your model after hyperparameter validation. Many datasets actually withhold the test set from programmers.

**Comment by AdamWP** [comment]
Thanks for taking the time to clarify.
